app/src/main/java/com/street35/booked/db/entity/BookCondition.java

```diff
@@ -0,0 +1,10 @@
+package com.street35.booked.db.entity;
+
+public enum BookCondition {
```

Comment:
Shall we not make this as a Dictionary object to store a little description also?

Reply:
Also, since BookCondition is used by one class only, we should prefer to keep this inside that class only